app/components/rights_component.html.erb

```diff
@@ -9,8 +9,13 @@
   <% end %>
   <% if license.present? %>
     <dt>License</dt>
-    <dd>
-      <%= license %>
-    </dd>
+    <dd><%= license %></dd>
+  <% end %>
+  <% if no_rights_information_present? %>
+    <dt>Attribution</dt>
+    <dd><%= default_attribution %></dd>
   <% end %>
 </dl>
+<div class="logoContainer">
+  <%= image_tag stanford_libraries_logo_url, alt: "Stanford Libraries logo", role: "presentation", class: "attributionLogo" %>
```

Comment:
Can we link to `<img src="https://stacks.stanford.edu/image/iiif/wy534zh7137/SULAIR_rosette/full/400,/0/default.jpg" alt="" role="presentation" class="mirador81">` instead?  This is what mirador does and it would prevent us from having to ship another copy of this rather large file. (chances are this one is already cached).

Reply:
@jcoyne I considered it, but I have no idea how stable that ☝🏻 link is. I guess as long as we're confident that druid will remain published and that filename won't change, it's fine. It at least increases parity among the players?

Reply:
I've never looked at it before but that druid is "SUL Logo 2015", which implies it might have been created for this purpose: https://purl.stanford.edu/wy534zh7137